src/main.js

```diff
@@ -68,8 +68,8 @@ function onUploadedData({ uploaded, length }) {
   }
 }
 
-function onSessionError({ message }) {
-  setStatus(`Error: ${message}`);
```

Comment:
we should have an action for this 

`store.dispatch(setErrorMessage(error.message))`